src/options/options.html

```diff
@@ -111,10 +113,10 @@ <h2 class="text-lg font-semibold text-gray-900">Select a workspace</h2>
             </div>
           </fieldset>
 
-          <div class="flex justify-end py-4">
+          <div class="flex flex-row-reverse gap-2 justify-start py-4">
             <button type="button"
-              class="inline-flex gap-x-2 items-center py-2.5 px-3.5 text-sm font-semibold text-white bg-indigo-600 rounded-md shadow-sm hover:bg-indigo-500 focus-visible:outline-2 focus-visible:outline-indigo-600 focus-visible:outline focus-visible:outline-offset-2"
-              @click="save">
+              class="inline-flex gap-x-2 items-center py-2.5 px-3.5 text-sm font-semibold text-white bg-indigo-600 rounded-md hover:bg-indigo-500 focus-visible:outline-2 focus-visible:outline-indigo-600 focus-visible:outline focus-visible:outline-offset-2 disabled:bg-indigo-500"
```

Comment:
Would it be possible to make the disabled state colors more explicit? From the video I didn’t notice it was disabled at first.

Reply:
Yep, sure thing!

Reply:
![CleanShot 2023-06-02 at 11 10 20@2x](https://github.com/orbit-love/orbit-browser-extension/assets/45462299/fb19a035-4792-4b19-aa89-0c6a70615bb2)